GPSSample selfMobility
RSUInfo[] rsuList
NeighborInfo[] neighborList
PedestrianInfo[] pedestrianList
ObjectInfo[] objectList